Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/RenamingPolicy.cs

```diff
@@ -87,13 +87,13 @@ public string CreateRemoteFileNameForMove(string sourceOperationTo, string sourc
                     "When using move as a source operation, you should always define a directory",
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/8c4346a7a0aded599281b906d6700bd7508cc79c/Frends.FTP.UploadFiles/Frends.FTP.UploadFiles/Definitions/RenamingPolicy.cs#L81
This method can be `static`.